src/config/server.rs

```diff
@@ -1,18 +1,18 @@
-use std::env::var;
+use std::{borrow::Cow, env::var};
 use tracing::info;
 
 const HOST_ENV_VAR: &str = "WORMHOLE_HOST";
 const PORT_ENV_VAR: &str = "WORMHOLE_PORT";
 const DEFAULT_HOST: &str = "127.0.0.1";
 const DEFAULT_PORT: u16 = 8080;
 
-pub fn get_host() -> String {
+pub fn get_host() -> Cow<'static, str> {
     if let Ok(host) = var(HOST_ENV_VAR) {
         info!("Using {} as the host since {} is set", host, HOST_ENV_VAR);
-        host
+        Cow::Owned(host)
     } else {
         info!("Using {} as the host", DEFAULT_HOST);
-        DEFAULT_HOST.to_owned()
```

Comment:
This `to_owned` call makes an unnecessary heap allocation that we'd rather avoid. Pretty small issue but also a pretty easy fix